src/components/dashboard.jsx

```diff
@@ -340,3 +344,113 @@ export const IFrameContainer = ({ src }) => {
         </>
     )
 }
+
+
+export const CardContainer = styled.div`
+    display: flex;
+    flex-flow: row;
+    flex: 1;
+    overflow: hidden;
+    margin-bottom: 1em;
+    gap: .35em;
+`
+export const CardContainerIcon = styled.div`
+    display: flex;
+    margin: auto;
+    width: 3em;
+    text-align: center;
+    align-items: middle;
+
+    @media screen and (max-width: 480px) {
+        display: flex;
+        margin: .75em 0 auto 0;
+        align-items: middle;
+    }
+`
+export const CardContainerInnerIcon = styled.div`
+    display: flex;
+    margin: auto;
+    font-size: 1em;
+`
+export const CardContainerText = styled.div`
+    display: flex;
+    flex-flow: column;
+    position: relative;
+    flex: ${props => props.flex ? props.flex : "1"};
+    overflow: hidden;
+
+    ${props => props.disabled ? `
+        opacity: 0.6;
+        user-select: none;
+        pointer-events: none;
+    ` : null}
+
+    @media screen and (max-width: 480px) {
+        flex-wrap: wrap;
+        width: 100%;
+        flex: 1;
+        display: ${props => props.mobileHide ? "none" : ""};
+    }
+`
+export const CardContainerInnerText = styled.div`
+    font-family: ${props => props.console ? 'monospace' : 'inherit'};
+    text-overflow: ellipsis;
+    width: 100%;
+    overflow: hidden;
+    white-space: nowrap;
+`
+export const CardContainerInputWrapper = styled.div`
+    display: flex;
+    flex-flow: row;
+    flex: 1;
+    gap: 1em;
+    flex-wrap: wrap;
+
+    @media screen and (max-width: 480px) {
+        flex-flow: column;
+        display: flex;
+        margin: auto;
+    }
+`
+export const CardContainerInput = styled.input`
+    &[type="text"], &[type="email"], &[type="date"], &[type="datetime-local"], &[type="number"] {
+        position: relative;
+        flex: 1;
+        border: 0;
+        border-bottom: 1px solid rgb(80,80,80);
+        outline: none;
+        padding: .15rem 0;
+        margin-bottom: .15rem;
+        background-color: white;
+        width: 100%;
+        user-select: inherit;
+    }
+    &:focus {
+        border-bottom: 1px solid rgb(255,75,157);
+    }
+    &:disabled {
+        background-color: inherit;
+        color: rgb(130, 130, 130);
+        border-bottom: 1px solid rgb(170,170,170)!important;
```

Comment:
`em`

Reply:
Same here, should I really use em when I only want 1px to be the border?